### PR TITLE
Fixed translation of "Compute Fibonacci numbers concurrently"

### PR DESCRIPTION
**Pull request by Ralinarar**
Fixed the title of the article and text in some places.

### CODE REVIEW

File: docs/book/zh/src/embed/rust/concurrent_fib.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 
-# 并行计算斐波那契数
+# 斐波那契数列的同时计算
```
Comment (alabulei1) [inline docs/book/zh/src/embed/rust/concurrent_fib.md:2]: concurrent 是专业术语，译为“并发”，比较合适。建议改成  斐波那契数列的并发计算

File: docs/book/zh/src/embed/rust/concurrent_fib.md [hunk 2]
```diff
@@ -10,11 +10,11 @@
 
 ## 概述
 
-在这个例子中，我们将演示如何使用 `wasmedge-sys` 中定义的对象和 API 来并行地计算斐波那契数。
+在这个例子中，我们将演示如何使用 `wasmedge-sys` 中定义的对象和 API 来同时计算斐波那契数。
```
Comment (alabulei1) [inline docs/book/zh/src/embed/rust/concurrent_fib.md:13]: 同上，建议更改为并发计算

**Review by alabulei1 — COMMENTED** [review]
@Ralinarar  Thanks for your contribution. Please check out my feedback.

**Comment by alabulei1** [comment]
@Ralinarar Could you see my comments?

**Comment by q82419** [comment]
No response after a month. Close.

**Pull request closed without merging**